actors/miner/src/lib.rs

```diff
@@ -933,6 +933,7 @@ impl Actor {
             &rew.this_epoch_baseline_power,
             &rew.this_epoch_reward_smoothed,
             &pwr.quality_adj_power_smoothed,
+            info,
```

Comment:
Is the point of adding this in just to optimize the aggregate_prove_commit method?  Or is there something nicer about this I am missing?

Reply:
I had to plumb sector size in for the check that deals fit in sector. Passing it from here avoids redundantly loading it later on.